evalml/tests/automl_tests/test_automl_search_classification.py

```diff
@@ -1252,6 +1253,8 @@ def test_automl_search_sampler_method(
     has_minimal_dependencies,
     caplog,
 ):
+    logger = logging.getLogger("evalml")
+    logger.setLevel(logging.DEBUG)
```

Comment:
Why is this necessary here? :0

Reply:
Aha, good catch! It was necessary before, since we run `assert "Could not import imblearn.over_sampling" in caplog.text` on line 1273 and that was previously logged at the debug level (so caplog didn't catch it). Since I switched that "Could not import" to log at a warning level, the caplog catches it without needing to increase the logging level. Will delete!